packages/core-components/src/components/Table/Table.tsx

```diff
@@ -154,7 +155,7 @@ const useFilterStyles = makeStyles<BackstageTheme>(
 
 export type TableClassKey = 'root';
 
-const useTableStyles = makeStyles<BackstageTheme>(
+const tableStyles = makeStyles<BackstageTheme>(
```

Comment:
I noticed it wasn't a hook, that's why I've renamed

Reply:
No this is very much a hook, stick to `useTableStyles` :p

It might be callable outside of React components in some situations, but conceptually it's still a hook.